Signal/src/ViewControllers/ContactShareViewHelper.swift

```diff
@@ -208,6 +203,7 @@ public class ContactShareViewHelper: NSObject, CNContactViewControllerDelegate {
             return
         }
 
+        UIUtil.applySignalAppearence()
```

Comment:
I'm not sure this is right, `ContactViewController`'s header has a dark background.  Therefore I was using"system appearance" aka `applyDefaultSystemAppearence` in `ContactViewController`.  

It looks like if we make this change, `ContactViewController` would have the wrong status bar appearance after editing a contact from it.

Reply:
Sorry, disregard this... just saw below.